### DISCUSSION TITLE
Doelmap bestaat al

### DISCUSSION THREAD

**Discussion by NNVV-EventsMadeEasy** [liedekef/events-made-easy#427] [Q&A] [answered]
Ik krijg deze foutmelding bij het installeren van de nieuwste versie:
Bezig met installeren van plugin vanuit geüpload bestand: events-made-easy.zip

Het pakket aan het uitpakken…

Plugin aan het installeren…

Doelmap bestaat al. /home/uxxxxp5126/domains/xxvereniging.nl/public_html/wp-content/plugins/events-made-easy/

Als ik deze map hernoem en doorga met installeren is dan de historie wel bewaard en toegankelijk? Ofwel: hoe nu verder?

**Comment by liedekef** [comment, marked as answer]
It is possible your WP is not allowing to overwrite that folder (maybe a permission issue). I never encountered this problem before, however what you still can do:
- disable the plugin
- delete the old plugin folder
- upload and install the new version
- reactivate the plugin

(disable/reactivate might not be needed, but to be sure ...).
As always with WP: make a DB backup just in case.
